pkg/lib/cockpit-components-onoff.jsx

```diff
@@ -55,7 +55,7 @@ export class OnOffSwitch extends React.Component {
         }
         var clickHandler = this.handleOnOffClick.bind(this, !this.props.state);
         return (
-            <div className="btn-group btn-onoff-ct">
+            <div id={this.props.id} className="btn-group btn-onoff-ct">
```

Comment:
All properties are described in the classes docstring above the definition. But since it's my only comment no need for reposting.

Reply:
Good point, thanks! I'll either fix that when having to do another force-push here, or in the follow-up PR for porting the shell.

Reply:
I had to force-push as I forgot one more rhel-7-6-distropkg special case in check-networking-basic. So fixed this as well.